<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BleSetting</class>
 <widget class="QWidget" name="BleSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-74</y>
        <width>521</width>
        <height>574</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="17" column="1">
          <widget class="QComboBox" name="threadCount">
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="fps">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="22" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="audioFormat"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Format:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>bitrate:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QComboBox" name="x264Preset"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Tune:</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QComboBox" name="x264Tune"/>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="bitrate">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>150</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>180</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>350</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>450</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>650</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;黑体&quot;;
color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>X264 Settings:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Preset:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;黑体&quot;;
color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>Encoder Settings:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>FPS:</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Profile:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="format">
           <item>
            <property name="text">
             <string>H264</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="res"/>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;黑体&quot;;
color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>Network Settings:</string>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>address:</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QLineEdit" name="address"/>
         </item>
         <item row="16" column="1">
          <widget class="QComboBox" name="keyFrameInterval"/>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Bitrate Mode:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QComboBox" name="bitrateMode"/>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>KeyFrame:</string>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Quality</string>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="qualityBar">
             <property name="styleSheet">
              <string notr="true"> QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 3px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
 }

 QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6d6d6d, stop:1 #5e5e5e);;
     border: 1px solid #5c5c5c;
     width: 8px;
     margin: -4px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 2px;
 }

 QSlider::add-page:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6d6d6d, stop:1 #5e5e5e);;
 }

 QSlider::sub-page:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ff9933, stop:1 #df8d3c);;
 }</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="qualityLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="14" column="1">
          <widget class="QComboBox" name="x264Profile"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;黑体&quot;;
color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>Audio Settings:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Device:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Channels:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Sample Rate:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Bitrate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="audioDevice"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="audioChannels"/>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="audioBitrate"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="audioSampleRate"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Format:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>kbps</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>HZ</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>ThreadCount:</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QCheckBox" name="enableBFrame">
           <property name="text">
            <string>enable B Frame</string>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>B Frame:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="text">
        <string>&amp;apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
